dashboard/config/routes.rb

```diff
@@ -41,8 +41,8 @@
     get "/congrats", to: "congrats#index"
 
     get "/incubator", to: "incubator#index"
-    get "/musiclab", to: redirect("/projectbeats", status: 302)
-    get "/projectbeats", to: "musiclab#index"
+    get "/musiclab", to: redirect("/projects/music/new")
```

Comment:
Should these go to the tutorial page?

Reply:
Good question.  We decided to redirect these to https://code.org/music.